mitodl/mitxpro
# Enroll user in edX course runs on order success

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #65 
Closes #195 

#### What's this PR do?
Enrolls the user in an edX course run when they complete an order

#### How should this be manually tested?
Complete an order with courses that exist on your edX instance and that your user is not currently enrolled in (it's easiest if you create a 100%-off coupon and apply it to avoid cybersource). After the order is complete, you should be able to see your course enrollment in your edX dashboard.

#### Any background context you want to provide?
Due to some edX complications, this enrollment is being done in "audit" mode. We'll change that enrollment mode in a separate PR
